pkg/metrics/metrics.go

```diff
@@ -239,9 +239,8 @@ var (
 	// endpoints, labeled by span name and status ("success" or "failure")
 	EndpointRegenerationTimeStats = NoOpObserverVec
 
-	// EndpointPropagationDelay is the delay between creation of local CiliumEndpoint
-	// and update for that CiliumEndpoint received through CiliumEndpointSlice.
-	// Measure of local CEP roundtrip time with CiliumEndpointSlice feature enabled.
+	// EndpointPropagationDelay is the propagation delay between creation or updation of local CiliumEndpoint
+	// received through CiliumEndpoints and CiliumEndpointSlice (if endpoint slice feature is enabled).
```

Comment:
```suggestion
	// EndpointPropagationDelay is the propagation delay between the creation (or update) of the local CiliumEndpoint
	// received through CiliumEndpoints and CiliumEndpointSlice (if the endpoint slice feature is enabled).
```

What does `received through CiliumEndpoints` mean exactly?

Reply:
:), should be received CiliumEndpoints (or CiliumEndpointSlice) event from k8s server.

Reply:
Reads better now, thanks!